CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.3
+
+<!-- generated comparing v5.0.0-beta.2..next -->
+
+_Aug 6, 2021_
+
+Big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- <!-- 28 --> ✨ Improve `jss-to-styled` codemod to support `createStyles` usage (#27578) @mnajdova
```

Comment:
I thought we were going to use passive here. We may also include that `<React.Fragments>` are being transformed now too after you rebase :)